src/components/Forms/SignIn/types.ts

```diff
@@ -0,0 +1,35 @@
+import { ApolloError, ServerError, ServerParseError } from '@apollo/client';
+import { SetContentType } from '../../AuthDialog/types';
+
+export interface QueryData {
+  login: {
+    userId?: string;
+    email?: string;
+    lastname?: string;
+    name?: string;
+    phonenumber?: string;
+  };
+}
+
+export interface QueryVars {
+  email: string;
+  password: string;
+}
+
+export interface CustomApolloError extends Omit<ApolloError, 'networkError'> {
+  networkError?: (Error | ServerParseError | ServerError | null) & {
+    result?: { errors?: { message?: string; status?: number | string }[] };
+    data: any;
```

Comment:
Any chance that it can be something else than `any` type?

Reply:
To be honest I can remove it because it is unnecessary in our case.